added dateOfPostEvent for kewen

PrincetonPoh opened the pull request.

Commit:
fixing issue where git push dont appear on github

PrincetonPoh committed:
Merge branch 'backendPrinceton' into sidebranchbackend

PrincetonPoh committed:
Merge pull request #1 from PrincetonPoh/sidebranchbackend

fixing issue where git push dont appear on github

Commit:
may have changed the sqlite file schema

Commit:
Merge branch 'backendPrinceton' of https://github.com/PrincetonPoh/esc-project into backendPrinceton

Commit:
remove wanghan files

Commit:
testing to see if the .sqlite3 works across branches

We-here committed:
added update user

We-here committed:
Create, get, delete, update posts

We-here committed:
Create, delete, update, getall posts

We-here committed:
Update escData.js

We-here committed:
Comment

PrincetonPoh committed:
refactoring to more modular code

We-here committed:
create new tables in database

We-here committed:
Merge branch 'backendPrinceton' of https://github.com/PrincetonPoh/esc-project into backendPrinceton

PrincetonPoh committed:
updated table

We-here committed:
Create Backup schema

We-here committed:
fix routes

We-here committed:
Posts API + change db schema

We-here committed:
Merge branch 'backendPrinceton' of https://github.com/PrincetonPoh/esc-project into backendPrinceton

PrincetonPoh committed:
updated users api calls to fit new DB

PrincetonPoh committed:
merge db

PrincetonPoh committed:
refactor wanghan code to to synch up. change the way we call query params for postman. updated postman

PrincetonPoh committed:
new postman with 'posts' api calls

We-here committed:
Search, delete of posts. Attend users

PrincetonPoh committed:
adding wanghan's changes

PrincetonPoh committed:
Merge branch 'backendPrinceton' of https://github.com/PrincetonPoh/esc-project into backendPrinceton

PrincetonPoh committed:
added new postman calls for posts

We-here committed:
Modify searchBasedOnPosts and updatePosts

We-here committed:
Merge branch 'backendPrinceton' of https://github.com/PrincetonPoh/esc-project into backendPrinceton

PrincetonPoh committed:
added comments api

PrincetonPoh committed:
attempt to setup jest

We-here committed:
added Princeton's changes for setting up testing framework

We-here committed:
Postman and delete, update userAttendList

PrincetonPoh committed:
added wanghan's changes

PrincetonPoh committed:
sync up postman

We-here committed:
Create Uuid

We-here committed:
PrimaryCode

tjianhui committed:
Small fixes?

tjianhui committed:
Small fixes?

PrincetonPoh committed:
add testing framework

Jukocross committed:
Added routes

PrincetonPoh committed:
added UUID, fixed most bugs, updated stable postman

PrincetonPoh committed:
Merge branch 'backendPrinceton' of https://github.com/PrincetonPoh/esc-project into backendPrinceton

PrincetonPoh committed:
changed start script back to node server.js

PrincetonPoh committed:
verify that postman is correct

PrincetonPoh committed:
clean up testing files

PrincetonPoh committed:
added fuse

We-here committed:
Search By title

We-here committed:
Change the sql database, add the primary code table

PrincetonPoh committed:
creating dummy jwt

PrincetonPoh committed:
adding wanghan's changes

PrincetonPoh committed:
jwt working locally using username

PrincetonPoh committed:
completed jwt implementation

PrincetonPoh committed:
added post tagging. added error codes for logins. added login api

PrincetonPoh committed:
added image api

We-here committed:
Add owner name to comment

We-here committed:
User tests and posts test

Jukocross committed:
Remove auth from certain API

PrincetonPoh committed:
added login with email verification

PrincetonPoh committed:
merge with wanghan's search by text updates

PrincetonPoh committed:
added email field in verifyEmail

Jukocross committed:
Updated routes

Jukocross committed:
Merged with v2.5/v2.6

Jukocross committed:
Added locations api

Jukocross committed:
Uploaded blueprint collection

We-here committed:
Add comment test

PrincetonPoh committed:
add email input

PrincetonPoh committed:
merge with yk

PrincetonPoh committed:
attempts to add a testing db + add some comments test case

We-here committed:
Create test database and clear it after testing

PrincetonPoh committed:
added readme

PrincetonPoh committed:
remove primary key tag for attendPosts table in db

PrincetonPoh committed:
rectify wrong db call for deletePostListsOfTheUser call

PrincetonPoh committed:
finally fixed port issue during test runs

PrincetonPoh committed:
adding dateOfPostEvent for kewen

Jukocross reviewed (approved):
Approved without testing by yk.

Jukocross committed:
To be revert

Jukocross committed:
To be revert

Jukocross committed:
Revert to 'Updated signup pages' hash

Jukocross committed:
Revert to dateOfPostEvent

Jukocross committed:
To be verified for latest version

We-here committed:
Changed tests a little bit

We-here committed:
Uploading test

PrincetonPoh committed:
small changes to testing for comments

tried to implement promises with knex but seems to have some issues so didnt push the changes

PrincetonPoh committed:
solving the merge conflict hell

We-here committed:
Try to add photo

We-here committed:
Merge branch 'backendPrinceton' of https://github.com/PrincetonPoh/esc-project into backendPrinceton

PrincetonPoh committed:
checked test cases
